run_nodocker.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I'd rather use `sh`, not eeeverybody has bash, esp. since we're not using any features of `bash` anyway

Reply:
The script does use one feature from bash:
```bash
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
It gets an error when run as `./run_nodocker.sh` on Ubuntu with the `/bin/sh` shebang. I don't know if there's any good way of finding the directory of the script with plain `sh`, and bash seems like a relatively harmless dependency.